annotator-core/src/main/java/edu/ucr/cs/riple/core/cache/downstream/DownstreamImpactCacheImpl.java

```diff
@@ -80,22 +82,34 @@ public DownstreamImpactCacheImpl(Context context) {
    */
   private ImmutableSet<Location> retrieveLocationsToCacheImpactsOnDownstreamDependencies(
       Context context, ModuleInfo moduleInfo) {
-    // Collect callers of public APIs in module.
+    ImmutableSet.Builder<Location> locationsToCache = ImmutableSet.builder();
+    // Collect public methods with non-primitive return types.
+    // Used to collect callers of each method.
```

Comment:
Should this comment be before line 90?

Reply:
Thanks for catching this. Fixed it in 15c8e1ec3b98b0e8b4ce4d26f2c2b83705e72206